app/views/insured/product_shoppings/show.html.slim

```diff
@@ -396,7 +396,7 @@ javascript:
                 <i class="fas fa-lg fa-square" data-toggle="tooltip" data-placement="top" data-container="body" title="${pvpTooltipValue}"></i> Premium Value Plan
               </span>
               <span class="${result.product.is_standard_plan ? (isPvpFeatureEnabled === 'true' ? 'standard-plan-indicator' : 'text-uppercase') : 'hidden'} ml-1" style="${isPvpFeatureEnabled === 'true' ? '' : 'color:green'}">
-                <i class="${isPvpFeatureEnabled === 'true' ? 'fas fa-lg fa-square' : 'fas fa-bookmark'}" data-toggle="tooltip" data-placement="top" data-container="body" title="${standardPlanTooltipValue}"></i> Standard Plan
+                <i class="fas fa-bookmark" data-toggle="tooltip" data-placement="top" data-container="body" title="${standardPlanTooltipValue}"></i> Standard Plan
```

Comment:
Ok, it's looking a bit more like the requirements doc/my info is out of date - I again would have expected a square here for standard plans where  `isPvpFeatureEnabled == true`. Assuming my info is old, feel free to disregard.